posthog/models/insight_comment.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+
+
+class InsightComment(models.Model):
+    insight: models.ForeignKey = models.ForeignKey(
+        "DashboardItem", related_name="comments", on_delete=models.CASCADE, null=False,
+    )
+    comment: models.CharField = models.CharField(max_length=400, null=True, blank=True)
```

Comment:
Maybe a TextField would work best here? We can avoid having a max length so users can add long comments.